backend/src/services/aws/s3/s3.client.ts

```diff
@@ -0,0 +1,12 @@
+import { S3Client } from '@aws-sdk/client-s3';
```

Comment:
move to backend/src/services/s3/s3.service.ts

Reply:
I think its ok to locate aws separately because it would contain a lot of logic. As we do with data access.

Reply:
agreed